src/passwordValidation.js

```diff
@@ -6,40 +6,40 @@ export const forbiddenPasswords = ["amG84h6yeQ", "mc9Q20pdjH", "jnT6Q2f8U5"];
  * @param {string} password
  * @returns {boolean}
  */
+
+const checkOrder = (password) => {
+  const ascendingNumbers = "123456789";
+  const descendingNumbers = "987654321";
+
+  // The following line extract of numbers in our password and return them as an array of numbers
+  const arrayOfNumbers = password.match(/([0-9])+/g);
+
+  for (let number of arrayOfNumbers) {
+    if (
+      (ascendingNumbers.includes(number) ||
+        descendingNumbers.includes(number)) &&
+      number.length > 1
+    )
+      return false;
+  }
```

Comment:
- I love how you broke down your work and I like it though you can still refactor this code. You have three loops together which may increase your time complexity in case the numbers are too much.
-  The two `includes` methods are first looping through the numbers to check if a particular number is present. Making a total of three loops altogether.